jsonnet/components/grafana.libsonnet

```diff
@@ -261,6 +275,51 @@ function(params)
                   },
                 ],
               },
+              {
```

Comment:
please add a comment about why we use a dedicated kube-rbac-proxy sidecar (e.g. because oauth-proxy doesn't support client TLS authn).